tests/selenium/src/lib.mjs

```diff
@@ -79,7 +83,7 @@ export async function waitBy(by, { optional = false, hidden = false, timeout = T
                 }
                 return false;
             }),
-            optional ? TIMEOUT_OPTIONAL : timeout
+            hidden ? HIDDEN_TIMEOUT : optional ? TIMEOUT_OPTIONAL : TIMEOUT_REQUIRED
```

Comment:
Please avoid nested ternaries and make new approach instead of `hidden` param (in the next branch).